Lombiq.Privacy/Migrations/Recipes/PrivacyConsentCheckboxSettings.recipe.json

```diff
@@ -15,7 +15,7 @@
         "Owner": "",
         "Author": "admin",
         "LiquidPart": {
-          "Liquid": "<span>I've read and agree to the site's <a href='/privacy-policy' target='_blank'>privacy policy</a>.</span>"
+          "Liquid": "<span>I've read and agree to the site's <a href=\"{{ '~/privacy-policy' | href }}\" target=\"_blank\">privacy policy</a>.</span>"
```

Comment:
What exactly does `{{ '~/privacy-policy' | href }}` do, and why did you change it in this PR?

Reply:
The URL prefix should be resolved so hard coded URL there is a bug, see [here](https://github.com/Lombiq/Lombiq-Tenants-Core/pull/3#discussion_r1102136992). I've changed it in this PR, because it had the same bug I've mentioned [here](https://github.com/Lombiq/Orchard-Vue.js/pull/75#issuecomment-1415664721) so we need this issue for the project.

Reply:
I was wondering what the ` | href` part means in that template.

Reply:
It's a built-in OC Liquid filter: https://docs.orchardcore.net/en/main/docs/reference/modules/Liquid/#href